app.py

```diff
@@ -4,9 +4,98 @@
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
+todos = [
+    {'id': 1, 'title': 'First Todo', 'description': 'do this tomorrow'},
+    {'id': 2, 'title': 'Second Todo', 'description': 'do this day after'},
+    {'id': 3, 'title': 'Third Todo', 'description': 'do this day after that'}]
+
+
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return 'Hello World! <a href=http://127.0.0.1:5000/todo> todos</a>'
+
+
+@app.route('/todo', methods=['GET'])
+@app.route('/todo/', methods=['GET'])
```

Comment:
Looks like what you wanted to always ignore trailing slashes. You can do this by setting `app.url_map.strict_slashes = False` and just defining the routes without the trailing slash